src/main/config.cpp

```diff
@@ -0,0 +1,75 @@
+#include <cstdio>
+#include <iostream>
+#include <sstream>
+#include "gflags/gflags.h"
```

Comment:
The convention is that system headers are included with `<>` (like iostream etc...) and local headers are only included using `""`. Since gflags is not part of hawk, I'd recommend that you use `<>`